src/psbt.rs

```diff
@@ -1,55 +1,48 @@
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
-use bitcoin::util::{bip32, psbt};
-use bitcoin::{Network, SigHashType};
-
-#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
-pub struct PsbtGlobalInfo {
```

Comment:
Why is this needed?

Reply:
This struct only had one field. From my understanding, the reason this was a struct, and not simply a variable, was because we wanted it to be returned from the `get_info(network)` implementation for the `psbt::Global` struct.

However, the bitcoin psbt module has been refactored to remove that `psbt::Global` struct, so it seemed like there should also be no need to hold a similar global struct here (since the original one it was created to work with no longer exists).

Reply:
Ah the `psbt::Global` concept is removed? Interesting. I didn't know that. Then it makes sense. :+1: